src/Map.jsx

```diff
@@ -53,6 +54,30 @@ function sortAlphaNumeric(a, b) {
 // make a copy of routes and sort
 const sortedRoutes = muniRoutesGeoJson.features.slice(0).sort(sortAlphaNumeric);
 
+class Stop {
```

Comment:
@EddyIonescu created class to make things easier

Reply:
This looks really elegant!